src/codeu/chat/util/CommandTokenizer.java

```diff
@@ -0,0 +1,68 @@
+package codeu.chat.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+final public class CommandTokenizer {
+
+	private String source;
```

Comment:
No, there isn't really a disadvantage. If you have a ton of similar variables you probably don't need to JavaDoc them all (otherwise it would get pretty messy), but in this case since you only have two I think that would be absolutely fine.